### ISSUE TITLE
Google+ API Shutdown

### ISSUE THREAD

**Issue by osvid** [gallir/Meneame#124]
According to Google, Google+ APIs will be shut down on March 7, 2019. This includes Google+ Sign-in and OAuth token requests with Google+ scopes. 

https://developers.google.com/+/api-shutdown
https://developers.google.com/+/integrations-shutdown

Entiendo que esto puede afectar al login en Menéame mediante Google. ¿Lo habéis comprobado o hecho una implementación alternativa?

Gracias.

**Comment by eusonlito** [comment]
Sí, lo tenemos más o menos localizado, pero estamos pendientes de si la actualización de la librería aplica los cambios en https://github.com/googleapis/google-api-php-client así como otros que ya están preparados https://github.com/laravel/socialite/pull/319/files pero aún tenemos que hacer las pruebas.

**Comment by eusonlito** [comment]
En cualquier caso, el nuevo repositorio es https://github.com/Meneame/meneame.net :)

**Comment by osvid** [comment]
Genial. No había visto el nuevo repo. Empiezo a seguirlo allí. Gracias.
